Cargo.toml

```diff
@@ -7,7 +7,9 @@ description = "A library for working with files and common text data encodings."
 license = "MIT"
 repository = "https://github.com/microsoft/file-content-rs"
 readme = "README.md"
+# At most 5 keywords
 keywords = ["encodings", "utf8", "utf16", "unicode", "BOM", "file-content"]
```

Comment:
Still see 6 keywords. Did you miss pushing a commit?

Reply:
Ah yep, I didn't pop my stash.